EIPS/draft_hardware_and_bandwidth_recs.md

```diff
@@ -0,0 +1,147 @@
+- Providing clear guidance for node operators with respects to the future
+
+Without a shared understanding of target hardware specifications:
+
+- Benchmark results lose significance due to inconsistent testing environments
+- Decision-making becomes challenging for implementation choices, as performance characteristics are heavily hardware dependent
+- Network participants lack clear guidance for hardware investments
+
+## Specification
+
+### Recommended hardware and bandwidth specifications
+
+| Node type | Storage | RAM | CPU Cores | CPU Single Thread/Multithread rating | Download/Upload speed |
+| -------- | -------- | -------- | --------| -------- |--------|
+| Full node     | 4TB     | 32GB    | 4 cores/8 threads  | 1000 / 3000 | 50 Mbps / 15 Mbps |
+| Attester      | 4TB     | 32GB    | 8 cores/16 threads | 3500 / 25000 | 50 Mbps / 25 Mbps |
+| Block builder | 4TB     | 32GB    | 8 cores/16 threads | 3500 / 25000 |  100 Mbps / 50 Mbps |
+
+### Recommended Prebuilds (Attester and Block builder)
+
+#### Minisforum UM790 Pro
+
+We recommend the Minisforum UM790 Pro with the following modifications:
+
+- Minisforum UM790 Pro
+- 8 cores
+- 16 threads
+- The barebones model will need to be updated to include 32GB RAM and 4TB storage
+- CPU Rating
+  - Single Thread: 3903
+  - Multithread: 30367
+
+#### ASUS NUC 14 Pro
+
+- 16 cores (6 performance + 8 efficiency cores + 2 Low power efficiency cores)
+- 22 threads
+- CPU Rating
+  - Single Thread: 3520
+  - Multithread: 25158
+
+### Recommended Prebuild (Full node)
+
+#### Raspberry Pi 5
+
+- 4 core
+- 4 threads
+- CPU Rating
+  - Single thread: 1487
+  - Multithred: 3428
+
+## Rationale
+
+### Storage
+
+For storage, we recommend the following SSDs:
+
+- Kingston KC3000
+- Kingston Fury Renegade
+- Samsung 990 Pro
+- Seagate Firecuda 530
+- Teamgroup MP44
+- WD Black SN850X
+
+In particular, we recommend purchasing NVMe M.2 instead of SATA as NVMe has higher throughput.
+
+As of January 2025, the storage requirements for all node types are dominated by the need to store block history.
+A 2TB disk can be used, however this is not recommended due to the rate of history growth and the uncertainty as to when EIP4444 will be complete.
+
+### RAM
+
+RAM/memory is dominated by state cache. As of January 2025, it is possible to run a full node with 16GB of RAM, however this has been known to not work with all combinations of EL and CL clients in the past.
+We therefore recommend 32GB.
+
+### Bandwidth
+
+- Statista states that as of January 2024:
+  - The global average download for broadband is 92 Mbps and the global average upload is 43 Mpbs.
+  - The global average download for mobile is 50 Mbps and 11 Mbps
+- GSMA report showing the state of mobile internet connectivity in 2024 shows that:
+  - The mobile upload speeds in Higher Income Countries (HIC) is about 18 Mbps
+  - The global average mobile download is 48 Mbps.
+
```

Comment:
GSMA link is https://www.gsma.com/r/wp-content/uploads/2024/10/The-State-of-Mobile-Internet-Connectivity-Report-2024.pdf?utm_source=website&utm_medium=button&utm_campaign=somic24

Paghe 39 for the upload speeds of HIC and page 5 for the 48 Mbps figure.

Statista link is: https://www.statista.com/statistics/896779/average-mobile-fixed-broadband-download-upload-speeds/

Reply:
Let me know how you would want these to be referred to in the document